support/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# try to fetch the latest version from github
+wget -O support.sh.new https://raw.githubusercontent.com/docker/libnetwork/master/support/support.sh
```

Comment:
LOL; second time this week I notice a "review comment pending" on github.. sorry for that

Reply:
I did not investigate the option of passing credentials to circle ci, yeah that would be definitely better, but this one is doing the job for now.

Reply:
Oh, but there's no need for that; I meant an auto build on Docker Hub; yes, it will rebuild on "any" pull request that's merged, but that shouldn't really matter.

I just set up one on my name;

<img width="1269" alt="screen shot 2018-06-21 at 23 25 25" src="https://user-images.githubusercontent.com/1804568/41761183-329e5a7e-75ab-11e8-88f3-60d6f63515ce.png">


I triggered it manually once; to make sure it builds;

```
docker run -v /var/run:/var/run --network host --privileged thajeztah/libnetwork-support
```

Now, wait for the next PR to be merged, and it should rebuild.